Orange/widgets/visualize/owpythagorastree.py

```diff
@@ -224,6 +229,8 @@ def set_tree(self, model=None):
             # if hasattr(model, 'meta_depth_limit'):
             #     self.depth_limit = model.meta_depth_limit
             #     self.update_depth()
+        self.send(ANNOTATED_DATA_SIGNAL_NAME,
```

Comment:
Is this needed? The "Selected data" signal is not sent here. I commented it out and it seems it still works.